bottle_server.py

```diff
@@ -36,14 +36,14 @@
 app = Bottle()
 default_app.push(app)
 
-import StringIO # NB: don't use cStringIO since it doesn't support unicode!!!
+import io # NB: don't use cStringIO since it doesn't support unicode!!!
```

Comment:
Can you add a comment to the README that this project now uses Python 3?

Reply:
Done